code/game/objects/items/weapons/material/material_armor.dm

```diff
@@ -44,6 +46,9 @@ Protectiveness | Armor %
 	STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/weapon/material/armor_plating/insert/Destroy()
```

Comment:
This override doesn't do anything.

Reply:
Without it, we got obj/item/weapon/material/armor_plating/insert was unable to be GC'd runtimes, I put it in to call QDEL_HINT_FINDREFERENCE, which returned no references.

But putting in the override stopped the runtime. I have no idea why.

Reply:
Unless there's another override for this subtype somewhere and this proc is compiled after it, this proc _literally_ does nothing. It just returns the result of the parent call. GC failures are generally down to other atoms are holding references, as well, which this wouldn't help with.

I would be extremely surprised and confused if this turned out to be definitely linked to GC fails.

Reply:
Tested this again, and having #define TESTING 1, set elsewhere was what stopped the runtime.
Again, no idea why.

Reply:
It sounds more like the GC failure is random or related to some holder atom and unrelated to either that define or this override. GC fails aren't really something to worry about unless they are systemic and constant. SS13 uses so many references and BYOND keeps so many internal ones that sometimes stuff just fails because it fails.